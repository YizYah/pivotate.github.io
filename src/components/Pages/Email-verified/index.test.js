import React from 'react';
import renderer from 'react-test-renderer';
import EmailVerified from '../Email-Verified';

test('Snapshopt Test', () => {
  const component = renderer.create(<EmailVerified />);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
