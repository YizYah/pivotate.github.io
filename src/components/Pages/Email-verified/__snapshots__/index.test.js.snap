// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshopt Test 1`] = `
<div
  className="text-center container full-width"
>
  <div
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderRadius": "10px",
        "margin": "auto",
        "marginBottom": "100px",
        "marginTop": "100px",
        "maxWidth": "769px",
      }
    }
  >
    <h2
      className="text-center"
    >
      <span
        className="headline headline--teal"
      >
         ACCOUNT ACTIVATED 
      </span>
    </h2>
    <p>
       Your email address has been verified.
    </p>
  </div>
</div>
`;
